spec/system/low_carbon_hub_installation_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+require 'dashboard'
+
+RSpec.describe "Low carbon hub management", :low_carbon_hub_installations, type: :system do
+  include_context "low carbon hub data"
+
+  let!(:school_admin) { create(:user, role: 'school_admin', school_id: school.id) }
+
+  context 'as school admin' do
+    before(:each) do
+      sign_in(school_admin)
+      visit root_path
+
+      @low_carbon_hub_api_instance = double("API")
+
+      click_on 'Manage Low carbon hub installations'
+      expect(page).to have_content("There are no Low carbon hub installations at the moment for this school")
+      click_on 'New Low carbon hub installation'
+    end
+
+    it 'I can add a low carbon hub installation' do
+      allow(LowCarbonHubMeterReadings).to receive(:new).and_return(@low_carbon_hub_api_instance)
+
+      expect(@low_carbon_hub_api_instance).to receive(:full_installation_information).with(rbee_meter_id).and_return(information)
```

Comment:
Could these be in the shared context too?

Reply:
Yes, but now with expect as the blowing up last test doesn't call these. Still, I'll switch to allow and move them.